.github/workflows/main.yml

```diff
@@ -1,6 +1,8 @@
 name: Detection workflow
 
 on:
+  schedule:
+    - cron: '0 18 * * *'
```

Comment:
This is 18:00 UTC = 20:00 CET. Did you choose this time based on some considerations? To me it looks good because it triggers after the business hours of most developers.